docs/setup_guides/hpc.rst

```diff
@@ -312,23 +405,30 @@ Main script
     # >>> client = Client(scheduler_file='~/scheduler.json')
 
     rm -f scheduler.json
-    mpirun --np 6 dask-mpi --nthreads 6 --memory-limit 22e9 --interface ib0 --local-directory /glade/scratch/$USER
+    mpirun --np 6 dask-mpi --nthreads 6 \
+        --memory-limit 22e9 \
+        --interface ib0 \
+        --local-directory /glade/scratch/$USER
```

Comment:
The local directory here will be an issue if someone copies this script naively.

Reply:
Not sure if it's appropriate for other site, but `TMPDIR` at NCAR points to `/glade/scratch/$USER`. Alternatively, you can set somewhere (in the module?) an environmental variable to use here, something like `PANGEO_LOCALDIR`?